boards/nano33ble_rev2/bootloaders/README.md

```diff
@@ -0,0 +1,23 @@
+Nano 33 BLE Sense Rev2 Bootloader Binaries
+===============================
+
+This folder contains pre-built bootloader binaries for the Nano 33 BLE Sense Rev2 board.
+The file names contain information about which bootloader it is and which
+address it is compiled for.
+
+Tock Bootloader Is Open Source
+------------------------------
+
+While we provide pre-built binaries for your convenience, you can build the
+bootloader yourself.
+
+```
+$ git clone https://github.com/tock/tock-bootloader
+$ cd tock-bootloader
+$ git checkout v1.1.0 # Choose a version, or use the latest commit
+$ cd boards/nano33ble_rev2-bootloader
```

Comment:
Are we going to do this, or will using the rev1 bootloader binaries work fine? I'm not inclined to update the bootloader if we don't have to.

Reply:
The bootloader binaries don't need to be changed since it's the same chip. The bootloader folder for the rev2 just contains the same files as the rev1.